.github/workflows/test.yaml

```diff
@@ -0,0 +1,49 @@
+name: Tests
+on:
+  push:
+
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+          - "3.8"
+
+    services:
+      dynamodb-local:
+        image: "amazon/dynamodb-local:latest"
+        ports:
+          - "8000:8000"
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: install deps
+        run: |
+          python -m pip install -e . -r requirements.txt
+
+      - name: run tests
+        run: make test
+
+  lint:
```

Comment:
This is running pretty quickly, but if it slows down we could cache `~/.cache/pre-commit`.